core/templates/dev/head/pages/base.html

```diff
@@ -210,6 +210,8 @@ <h2>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/IdGenerationService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/ValidatorsService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/services/HtmlEscaperService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/TranslationFileHashLoaderService.js"></script>
```

Comment:
Is it required here (line 214)?

Reply:
ok i'll update

Reply:
@jervis446, Have you checked it? as it's required in Library page so Undo this (Reason behind E2E test failures).